core/app/[locale]/(default)/page.tsx

```diff
@@ -89,6 +90,14 @@ export default async function Home({ params }: Props) {
         products={bestSellingProducts}
         title={t('BestSellingProducts.title')}
       />
+
+      <Subscribe
+        // @ts-expect-error - We don't have an action ready for this component. TODO: add action
+        action={null}
```

Comment:
Should this action be defined before we merge this?

Reply:
Instead of passing `null` with a `@ts-expect-error` and the very least we should implement a stub for the action that shows an alert or logs something to the console server-side, no?